jormungandr/src/settings/start/config.rs

```diff
@@ -197,11 +187,11 @@ pub fn default_interests() -> BTreeMap<Topic, InterestLevel> {
     BTreeMap::from_iter(vec![
         (
             Topic(topic::MESSAGES),
-            InterestLevel(poldercast::InterestLevel::Low),
+            InterestLevel(poldercast::InterestLevel::new(1)),
```

Comment:
Maybe define our own consts with application-defined meanings of low and normal.

Reply:
Agree, this is indeed reworked in https://github.com/input-output-hk/jormungandr/pull/3178. Now I see that the splitting of the PRs is starting to make some confusion, not sure what the best method would be 🤔